Gemfile.lock

```diff
@@ -361,6 +390,7 @@ DEPENDENCIES
   rails!
   rake (>= 10.3)
   redcarpet (~> 3.2.3)
+  redis (~> 3.0)
```

Comment:
I'm not sure that'll work here, since we conditionally require the adapter based on the configuration, but Action Cable will always end up being required. I'm concerned that this will cause a large number of existing apps to be unable to upgrade by default, if Redis isn't already installed on the machine.